libs/drawer/src/index.tsx

```diff
@@ -47,7 +47,31 @@ const Drawer = ({
           testId="drawerCloseButton"
           top="8px"
         />
-        <Stack direction="column" height="100%" spacing={4} sx={{ '&>*': { bg: drawerBackground } }}>
+        <Stack
+          direction="column"
+          height="100%"
+          overflow="auto"
+          spacing={4}
+          sx={{
+            '::-webkit-scrollbar': {
+              width: '14px',
+            },
+            '::-webkit-scrollbar-thumb': {
+              boxShadow: 'inset 0 0 14px 14px white',
+              borderColor: 'transparent',
+              borderStyle: 'solid',
+              borderWidth: '0 0 0 11px',
+            },
+            '::-webkit-scrollbar-track': {
+              boxShadow: 'inset 0 0 14px 14px',
+              borderColor: 'transparent',
+              borderStyle: 'solid',
+              borderWidth: '0 0 0 11px',
+              color: 'dolbySecondary.500',
+            },
```

Comment:
Before:

<img width="1063" alt="Screen Shot 2023-01-09 at 2 14 20 pm" src="https://user-images.githubusercontent.com/59328090/211235706-bbc9c7da-c752-4fab-bbd0-7bba28fe4162.png">

---

After:

<img width="1059" alt="Screen Shot 2023-01-09 at 2 12 41 pm" src="https://user-images.githubusercontent.com/59328090/211235723-9f2ec782-f529-4407-b675-9d7ae71c171b.png">


N.B. this isn't actually in the designs, but I've added it as a thematically appropriate temporary solution for overflow issues.

Reply:
Please double confirm with designer

Reply:
https://www.figma.com/file/mpGO2yc1EBeDKgcvhmm5CA?node-id=6885:29346#340922841